go/vt/schema/ddl_strategy.go

```diff
@@ -116,6 +118,9 @@ func ParseDDLStrategy(strategyVariable string) (*DDLStrategySetting, error) {
 	if _, err := setting.RetainArtifactsDuration(); err != nil {
 		return nil, err
 	}
+	if _, err := setting.ForceCutOverAfter(); err != nil {
+		return nil, err
+	}
```

Comment:
Where do we limit this to the vitess strategy? Maybe I missed it.

Reply:
Fixed. Now erroring when not in `vitess` strategy.